themes/default/content/docs/intro/console/audit-logs.md

```diff
@@ -47,14 +47,14 @@ also filter logs by a particular user by selecting their profile picture.
 ## Automated Export
 
 {{% notes "info" %}}
-[Contact us]({{< relref "/contact" >}}) if you would like to activate this feature for your enterprise organization.
+This feature is only available to some Pulumi Enterprise editions. If you don't see it in your organization, [contact us]({{< relref "/contact?form=sales" >}}).
```

Comment:
if we are gonna select sales for them, it might be cool to indicate that in the link text, so `contact sales` instaed